### ISSUE TITLE
Show current ticket price for an event

### ISSUE THREAD

**Issue by karthikb351** [hasgeek/events#137]
Since https://github.com/hasgeek/boxoffice/pull/167 has been merged we can specify the `items_ids` we want to render in the widget, and how to order them.

A side effect of this is that we can use boxoffice's `/order/kharcha` API to get the current price for a ticket and render it.

This will require us flagging a specific item id as the _primary_ one, which is usually the most common ticket people may purchase.

**Comment by mediatinker** [comment]
I would say as an option to the most common item, the lowest price option (a one-day ticket) woudl allow us to say something like "tickets from X Rs."

**Comment by karthikb351** [comment]
That's a good point - but you'll still need to manual mark which item that is since t-shirts/workshops might be lower priced so we can't rely on just cheapest's item.

**Comment by mediatinker** [comment]
True. I forgot that we might be separating out the add-ons (which makes so much sense).
